src/xunit.runner.uap/Helpers.cs

```diff
@@ -15,5 +16,14 @@ internal static class Helpers
             return await folder.CreateFileAsync(test, CreationCollisionOption.ReplaceExisting);
         }
 
+        public static StreamWriter GetStreamToFileInLocalStorage(string pathInLocalStorage)
+        {
+            string appSharedPath = Directory.GetParent(Windows.Storage.ApplicationData.Current.LocalFolder.Path).FullName;
```

Comment:
>GetParent [](start = 45, length = 9)

why getting the parent?

Reply:
In order to access the SharedLocalFolder (the package root folder) through `ApplicationData` we need special permissions, so I'm getting the LocalFolder which will be something like: `C:\Users\safern\AppData\Local\Packages\5cd54353-3ed7-4a6e-a72f-db349f28867c_v52bfwc2c21ha\Local\` so I will get the parent which is the root folder and then create the file where the Application Launcher expects it. I could change it to be under the `LocalFolder` also.